src/coreclr/inc/readytorun.h

```diff
@@ -397,6 +397,10 @@ enum ReadyToRunHelper
     // Floating point ops
     READYTORUN_HELPER_DblRem                    = 0xE0,
     READYTORUN_HELPER_FltRem                    = 0xE1,
+
+    // These two helpers can be removed with next major version bump
```

Comment:
```suggestion
    // These two helpers can be removed once MINIMUM_READYTORUN_MAJOR_VERSION is 10+
```

Reply:
Should there be a comment immediately above MINIMUM_READYTORUN_MAJOR_VERSION telling people to do a grep for such cases, so we don't miss out on removing them later?

Reply:
Basically something like:
> // If you update this, ensure you `git grep MINIMUM_READYTORUN_MAJOR_VERSION` and handle pending work